### DISCUSSION TITLE
WLW196i no power readings during manual dhw charging

### DISCUSSION THREAD

**Discussion by popopirat** [emsesp/EMS-ESP32#1961] [General]
Hi all,
I am reading the current power consumption of the heatpump with the custom sensor:
Device ID 0x08
	
TypeID 0x4AA
	
Addition 0
	
Type USHORT

This works nicely when the heatpump is running based on the internal scheduling during the day.
However if I manually start a hot water charge with "Thermostat-> dhw charge"
I have 0 W readings even though the heatpump is running. I can see this in my houses power consumption.
Edit: okay, I figured out that the switch seems to start the iauxiliary heater instead of the heatpump which turns off after a couple of minutes.

Furthermore, a hot water charge using the "Boiler -> dhw charging " switch is not possible.

Does anyone have a solution for the problem?

**Comment by proddy** [comment]
maybe @UlfHRO or @mheyse based on comments in #1717

**Reply by UlfHRO** [reply]
Hi,
power | 0x08 | 0x4AA | 0 | UINT16
works fine for my WLW186 in all operation modes...

Extra dhw charge can be started with "dhw charge" in the thermostat type...

Regards,
Ulf
